views/admin/admin-page.php

```diff
@@ -56,18 +56,15 @@
 </div><!-- .landing -->
 
 	<?php else : ?>
-		<div class="wpcom-connect">
-			<h1 title="Boost traffic, enhance security, and improve performance."><?php esc_html_e( 'Boost traffic, enhance security, and improve performance.', 'jetpack' ); ?></h1>
+		<div id="jump-start-area" class="j-row">
+			<h1 title="Please Connect Jetpack."><?php esc_html_e( 'Please Connect Jetpack', 'jetpack' ); ?></h1>
 			<div class="j-row">
-				<div class="j-col j-sm-12 j-md-8 j-lrg-7 connect-desc">
-					<p><?php _e('Jetpack connects your site to WordPress.com to give you traffic and customization tools, enhanced security, speed boosts, and more.', 'jetpack' ); ?></p>
-					<p><?php _e('To start using Jetpack, connect to your WordPress.com account by clicking the button (if you don’t have an account you can create one quickly and for free).', 'jetpack' ); ?></p>
-				</div>
-				<div class="j-col j-sm-12 j-md-4 j-lrg-5 connect-btn">
+				<div class="j-col j-sm-12 j-md-8 connect-desc" style="float:none; margin:0 auto; text-align:center; padding-bottom:20px;">
```

Comment:
Fixed in https://github.com/Automattic/jetpack/commit/23c4a8cfbcd1a95b64ad5cbe181836129b9839c6